src/hooks/useAccountRiskCheck.ts

```diff
@@ -11,11 +11,12 @@ export default function useAccountRiskCheck(account: string | null | undefined)
       const riskCheckLocalStorageKey = `risk-check-${account}`
       const now = Date.now()
       try {
+        // Check local browser cache
         const storedTime = localStorage.getItem(riskCheckLocalStorageKey)
         const checkExpirationTime = storedTime ? parseInt(storedTime) : now - 1
         if (checkExpirationTime < Date.now()) {
           const headers = new Headers({ 'Content-Type': 'application/json' })
-          fetch('https://screening-worker.uniswap.workers.dev', {
+          fetch('https://api.uniswap.org/v1/screen', {
```

Comment:
is there a reason this isn't using `react-query`?

Reply:
it probably should but then again... the added middleware overhead will be annoying xD